src/python/review/inspectors/flake8/flake8.py

```diff
@@ -63,10 +64,21 @@ def parse(cls, output: str) -> List[BaseIssue]:
 
     @staticmethod
     def choose_issue_type(code: str) -> IssueType:
+        # Handling individual codes
         if code in CODE_TO_ISSUE_TYPE:
             return CODE_TO_ISSUE_TYPE[code]
 
-        code_prefix = re.match(r'^([a-z]+)\d+$', code, re.IGNORECASE).group(1)
+        regex_match = re.match(r'^([a-z]+)(\d+)$', code, re.IGNORECASE)
```

Comment:
Maybe consider tweaking regex?
regex_match = re.match(r'^([A-Z]+)(\d{1})\d*$', code, re.IGNORECASE) 
In the case above the second group will capture the first code number and can be added to CODE_PREFIX_TO_ISSUE_TYPE as a key (as currently groups are divided by hundreds in ranges). In that case no need to create WPS_RANGE_TO_ISSUE_TYPE and make ‘for’ cycle through the items while handling WPS issues.